src/modules/dashboard/components/Dialogs/RemoveSafeDialog/RemoveSafeDialog.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { FormikProps } from 'formik';
+
+import Dialog, { DialogProps } from '~core/Dialog';
+import { ActionForm } from '~core/Fields';
+
+import { Colony } from '~data/index';
+import { ActionTypes } from '~redux/index';
+import { WizardDialogType } from '~utils/hooks';
+
+import { Safe } from './types';
+import DialogForm from './RemoveSafeDialogForm';
+
+export interface FormValues {
+  safeList: string[];
+}
+
+interface CustomWizardDialogProps {
+  prevStep: string;
+  colony: Colony;
+}
+
+type Props = DialogProps & WizardDialogType<object> & CustomWizardDialogProps;
+
+const displayName = 'dashboard.RemoveSafeDialog';
+
+const safes: Safe[] = [
```

Comment:
I think it would be good to put a comment here for this mock data, so, that we know to remove it in future.